### ISSUE TITLE
README Controller tests should explain need for "include Devise::Test::IntegrationHelpers" in rails 5

### ISSUE THREAD

**Issue by yusukemorita** [plataformatec/devise#4819]
## Precheck

- Do not use the issues tracker for help or support, try Stack Overflow.
- For bugs, do a quick search and make sure the bug has not yet been reported
- Finally, be nice and have fun!

## Environment

- Ruby **[2.5.0]**
- Rails **[5.1.5]**
- Devise **[4.4.1]**

## Current behavior

In the Controller tests section of the README, it is stated that `Devise::Test::ControllerHelpers` needs to be included for devise to work in tests. The following sample code is also supplied.

```
class PostsControllerTest < ActionController::TestCase
  include Devise::Test::ControllerHelpers
end
```

However, from Rails 5 controller tests no longer belong to `ActionController::TestCase`, and now belong to ` ActionDispatch::IntegrationTest`. Therefore, the above example code is not valid for rails 5, and should look something like this instead.

```
class PostsControllerTest < ActionDispatch::IntegrationTest
  include Devise::Test::IntegrationHelpers
end
```

I think it would be easier for developers using rails 5 if there was a short passage explaining this, and I would be delighted to create a pull request.

**Comment by tegon** [comment]
Yeah, Rails 5 doesn't have controller tests anymore. We do have a session on the readme for integration tests too https://github.com/plataformatec/devise#integration-tests
But we aren't saying that Rails 5 comes with them by default. I guess it makes sense to add a note about this in the controller tests section.

If you're willing to open a PR for this, I'd be happy to accept it.

Thanks!

**Comment by yusukemorita** [comment]
Thanks for your quick comment @tegon , I've opened a PR :)
https://github.com/plataformatec/devise/pull/4820

**Comment by tegon** [comment]
Closed via #4820
